ophyd/ophydobj.py

```diff
@@ -61,8 +72,15 @@ def root(self):
 
     @property
     def report(self):
+        '''A report on the object.'''
         return {}
 
+    @property
+    def event_types(self):
+        '''Events that can be subscribed to via `obj.subscribe`
+        '''
+        return self._pub_event_types
```

Comment:
I don't think it's a bad thing to expose it - but it's another thing that _may_ clash with any of the 1000 devices that are out there in the beamline configs.